### ISSUE TITLE
Implement CLI library

### ISSUE THREAD

**Issue by ThibaultNocchi** [HarvestersCrew/CorpusHarvester#13]
We should try implementing a real CLI library to avoid a large part of the hassles. See those:

- https://github.com/adishavit/argh
- https://github.com/taywee/args
- https://github.com/CLIUtils/CLI11
- https://github.com/jarro2783/cxxopts

(found them here: https://github.com/fffaraz/awesome-cpp#cli)

**Comment by RegisGraptin** [comment]
Ok I implemented the argparse library (https://github.com/p-ranav/argparse).
I had some problems with the libraries that you refer. I just post an issue in 'argh' and 'CLI11'. If the issues are resolved and the current library is not suitable, I will change the library.

Now, you can create a corpus with spaces (like "star wars"). 
Here is a very light documentation (https://github.com/HarvestersCrew/CorpusHarvester/tree/prototype/cli). In the future, I will add more features.
So basically, now you can do : `bin/cli --create star wars`

However with the argparse library we have a problem in the case we need to have more parameters with spaces. Indeed, only one parameter can have multiple spaces, and it has to be at the end of the command.
